RLTSquare/CustomEmailTemplate/composer.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "rlt-square/module-custom-email-template",
+  "version": "1.0.0",
+  "description": "N/A",
+  "type": "magento2-module",
+  "require": {
+    "magento/framework": "*"
```

Comment:
Undeclared module dependency - nice touch on adding this file ... but ... your code uses Magento\Store\Model\ScopeInterface and Magento\Store\Model\StoreManagerInterface but module.xml has no dependency declared on store module.